tests/e2e/subbyte_types/BUILD.bazel

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 The IREE Authors
```

Comment:
2024

Reply:
This is not fixed yet.

Reply:
Apologies, it was lost in my internal manipulation of those files...